kafka-client/src/test/java/dev/responsive/kafka/async/AsyncProcessorIntegrationTest.java

```diff
@@ -261,7 +260,7 @@ public void shouldExecuteMultipleMixedAsyncProcessorsNoCaching() throws Exceptio
                       sleepForMs(1L);
                       return new SimpleProcessorOutput<>(val.getValue() + "-S1:" + newSum, newSum);
                     },
-                    ResponsiveKeyValueParams.fact(asyncStore1),
+                    ResponsiveStores.factStore(asyncStore1),
```

Comment:
I ran a full end-to-end test to make sure async works without a Responsive storage backend by plugging in `StorageBackend.NONE` and flipping all the stores from a Responsive store supplier to a regular Streams store supplier.

Ultimately I didn't feel it was worth parametrizing the test on this since these async tests have fairly long runtimes and there's not much value add, so I didn't commit this change. The problem is that the existing tests are all fairly heavy. I do think that once we can upgrade to 4.0 and get async working with the DSL, it could be worth running a short test with `NONE`, because I was going to write a large number of much lighter tests for the spread of DSL operators, so that would be a good opportunity for this as well